updater/site/site_handler.py

```diff
@@ -23,6 +24,8 @@ def get_handler(url: str, game_version: GameVersion,
         return WoWInterface(url, game_version)
     elif GitHub.handles(url):
         return GitHub(url)
+    elif GitHubRelease.handles(url):
+        return GitHubRelease(url)
```

Comment:
Also a nitpick, could have dodged changing `GitHub`'s handler if we just check the release style first
```suggestion
    elif GitHubRelease.handles(url):
        return GitHubRelease(url)
    elif GitHub.handles(url):
        return GitHub(url)
```

Reply:
I've been thinking a bit more about this and while it is indeed true that this would allow the default implementation to remain for `GitHub.handles`, it would also make `GitHub.handles` incorrectly report `True` for urls that it can not handle correctly. When you append `/releases` to the url the code in `GitHub.get_addon_name` will misbehave and cause the addon download to no longer work.

Reply:
Sure enough, it was "broken" before by that definition. Your reasoning is fine.